libs/pmp-web/shared/ui-statistics-overview-table/src/lib/statistics-overview-table/statistics-overview-table.component.scss

```diff
@@ -34,3 +34,9 @@ mat-cell {
   border-radius: 50%;
   margin: auto;
 }
+
+.content-loader {
```

Comment:
it shouldn't be duplicated

Reply:
https://angular.io/api/router/NavigationExtras#state

Reply:
So where should it be placed? In global assets directory?

Reply:
Yes, until we confirm or delegate to someone to implement that idea with styles.